src/test_fixture/transport/mod.rs

```diff
@@ -1,13 +1,21 @@
-pub mod network;
+mod network;
```

Comment:
This file is empty!

Reply:
which file? network.rs has something it seems